packages/modules/viewer-react/src/hooks/useUiProviders.tsx

```diff
@@ -3,20 +3,35 @@
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
 
+import { TreeWidgetUiItemsProvider } from "@bentley/tree-widget-react";
 import { UiItemsManager, UiItemsProvider } from "@bentley/ui-abstract";
 import { useEffect } from "react";
 
-export function useUiProviders(uiProviders?: UiItemsProvider[]): void {
+import { ItwinViewerUi } from "../types";
+
+export function useUiProviders(
+  customUiProviders?: UiItemsProvider[],
+  defaultUiConfig?: ItwinViewerUi
```

Comment:
dont you need to pass in the defaultUiConfig in IModelLoader?

Reply:
todo: later on need to figure out how to correctly pass options to these uiProviders